fs_attachment_s3_migration/models/ir_attachment.py

```
@@ -0,0 +1,305 @@
+                force_db_config,
+            )
+
+        _logger.info(
+            "Starting migration enqueue for storage %s (batch_size=%d, max_batches=%s)",
+            storage_code,
+            batch_size,
+            max_batches or "unlimited",
+        )
+
+        while True:
+            cursor_domain = domain + [("id", ">", last_id)]
```

Ruff (0.14.10)

Consider `[*domain, ("id", ">", last_id)]` instead of concatenation

Replace with `[*domain, ("id", ">", last_id)]`

(RUF005)